spec-main/api-net-spec.ts

```diff
@@ -628,7 +628,7 @@ describe('net module', () => {
       })
     })
 
-    describe('webRequest', () => {
+    xdescribe('webRequest', () => {
```

Comment:
I added a `net.isNetworkServiceEnabled` JS flag in https://github.com/electron/electron/pull/19094, which can be used to conditionally skip the tests.

Reply:
Yup plan to use that once merged 👍